game/scripts/vscripts/modifiers/modifier_boss_capture_point.lua

```diff
@@ -0,0 +1,155 @@
+modifier_boss_capture_point = class(ModifierBaseClass)
+
+local particleDirectory = "particles/capture_point_ring/"
+
+function modifier_boss_capture_point:IsPurgable()
+  return false
+end
+
+function modifier_boss_capture_point:IsHidden()
+  return true
+end
+
+-- Looks up the name on self and cleans up the particle and index
+function modifier_boss_capture_point:DestroyParticleByName(particleName)
+  local particleIndex = self[particleName]
+  if particleIndex then
+    ParticleManager:DestroyParticle(particleIndex, false)
+    ParticleManager:ReleaseParticleIndex(particleIndex)
+    self[particleName] = nil
+  end
+end
+
+function modifier_boss_capture_point:StartInProgressParticle()
+  if not self.captureInProgressEffect then
+    self.captureInProgressEffect = ParticleManager:CreateParticle(particleDirectory .. "capture_point_ring_capturing.vpcf", PATTACH_ABSORIGIN, self:GetParent())
+    ParticleManager:SetParticleControl(self.captureInProgressEffect, 9, Vector(self.radius, 0, 0))
+  end
+  ParticleManager:SetParticleControl(self.captureInProgressEffect, 3, self:GetColor())
+end
+
+function modifier_boss_capture_point:StartClockParticle()
+  if not self.captureClockEffect then
+    self.captureClockEffect = ParticleManager:CreateParticle(particleDirectory .. "capture_point_ring_clock.vpcf", PATTACH_ABSORIGIN, self:GetParent())
+    ParticleManager:SetParticleControl(self.captureClockEffect, 9, Vector(self.radius, 0, 0))
+    -- Controls how much of the dial to spawn. 1 is the full circle
+    ParticleManager:SetParticleControl(self.captureClockEffect, 11, Vector(0, 0, 1))
+  end
+  ParticleManager:SetParticleControl(self.captureClockEffect, 3, self:GetColor())
+end
+
+function modifier_boss_capture_point:GetColor()
+  local neutralColor = Vector(255, 150, 0)
+  local radiantColor = Vector(0, 162, 255)
+  local direColor = Vector(5, 155, 255)
```

Comment:
The dire color is wrong in think. This is should be (241, 5, 5 ) or some sort of red